rmgpy/rmg/pdepTest.py

```diff
@@ -162,19 +162,14 @@ def setUp(self):
         self.pdepnetwork.index = 1
         self.pdepnetwork.explored = []
 
-    
-    def test_SS_solver(self):
```

Comment:
But pdep.py still had the method, doesn't it? (or was the method itself removed as well?)
If we're still using the original method somewhere, lets keep/relocate the test for it?

Reply:
So the method exists, but the problem is that pdepTest doesn't have appropriate objects to test it directly after changes made in solve_SS_network. test_flux_filter does test it though, so it's still under coverage.
I invested sometime trying to figure out how to make it work, but given it's already under reasonable coverage (get_rate_filtered_products isn't a complicated function), it didn't seem worth it.